library/Recurly/V3/API/Types/Transaction/CvvCheck.hs

```diff
@@ -3,42 +3,42 @@ module Recurly.V3.API.Types.Transaction.CvvCheck where
 import Recurlude
 
 -- Yes recurly these names are very useful
```

Comment:
😆 

Do these short names (like `D`) map to anything meaningful, or are they just codes that Recurly uses? If they are meaningful, that meaning could be encoded in the constructor names.

Reply:
They probably have meaning but it's currently unused, the next time we use them and find their meaning we can change it 😝